paddle/fluid/platform/nccl_helper.h

```diff
@@ -41,6 +41,11 @@ inline ncclDataType_t ToNCCLDataType(std::type_index type) {
   }
 }
 
+// NOTE(minqiyang): according to the ncclGroupEnd documentations:
```

Comment:
Well, I think we can assume people who develop paddlepaddle with this file is familiar with the NCCL natives.

Reply:
To avoid this bug happens again, I leave these comments.